tm-scala.yaml

```
@@ -24,6 +24,7 @@ commands:
       - run:
           name: Preparing cache files
           command: |
+            mkdir -p ~/.ivy2/cache
             rm -fv ~/.ivy2/.sbt.ivy.lock
             find ~/.ivy2/cache -name "ivydata-*.properties" -print -delete
             find ~/.sbt        -name "*.lock"               -print -delete
```

I previously solved this issue in Piranha by ignoring failures in the commands, rather than creating the directory first: https://github.com/teikametrics/piranha/blob/93669073066ac0d741597b9b8ea7d7b2fc82dd17/.circleci/config.yml#L34-L38

I guess not really much difference, except it avoids the folder creation IO ¯\\\_(ツ)\_/¯

```suggestion
          command: |
            rm -fv ~/.ivy2/.sbt.ivy.lock || true
            find ~/.ivy2/cache -name "ivydata-*.properties" -print -delete || true
            find ~/.sbt        -name "*.lock"               -print -delete || true
```